policy/policy.sample.json

```diff
@@ -0,0 +1,70 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1", # names can be same for multiple tenants thus use uuid.
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants.
+            "Associations": [ # can attach multiple tenants associations in case you want blanket policy support.
```

Comment:
Would "AppliedTo" be more descriptive?

Because all sorts of things are "associated" with the policy. For example, both source and destination are somehow associated. So is the tenant, the cluster, etc. In our context the word "Associations" may be a little too generic. Our use case is very specific: We want to tell the system where to apply the policies.